src/main/java/jalau/at18/azul/MixedBag.java

```diff
@@ -0,0 +1,12 @@
+package jalau.at18.azul;
+
+import java.util.*;
+
+public class MixedBag {
+
+    public ArrayList<Character> mixColors(ArrayList<Character> simpleBag) {
```

Comment:
Is there a case where we need to have both cases available? colors mixed and not mixed? if we only need mixed colors, this method could be part of bag and `MixedBag` can be removed, so a Bag always mix its colors

Reply:
efectively, Bag class generates a bag full of tiles, in order, there is a method now included to shuffle the bag and return a mixed bag for further use in the game, Thank You Jose for your comments.